src/Components/WebAssembly/BlazorManifest/src/WorkloadManifest.json.in

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This file needs to go to dotnet/sdk (so the versioning should happen there)

Reply:
Could you elaborate on this? I'm missing something fundamental about how this gets shipped.

Reply:
I think we probably need to discuss with @dsplaisted where this should be built into. The spec only mentions the destination https://github.com/dotnet/designs/blob/9e94ff32ea43418310c0fc77eb3703492856b364/accepted/2020/workloads/workload-manifest.md#advertising